src/org/opendatakit/briefcase/export/ExportConfiguration.java

```diff
@@ -449,7 +447,7 @@ public Path getExportMediaPath() {
   }
 
   public Optional<String> getExportFileName() {
-    return exportFileName;
+    return exportFileName.map(filename -> filename.endsWith(".csv") ? filename : filename + ".csv");
```

Comment:
Might there be upper or mixed case “csv”?

Reply:
Good point. I’ll take that into account